README.md

```diff
@@ -9,75 +9,145 @@ EditURL = "<unknown>/README.jl"
 [![Build Status](https://travis-ci.com/gridap/GridapMakie.jl.svg?branch=master)](https://travis-ci.com/gridap/GridapMakie.jl)
 [![Coverage](https://codecov.io/gh/gridap/GridapMakie.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/gridap/GridapMakie.jl)
 
-# This package is experimental
+## Overview
 
-Things can and will break without deprecation or warning
+The visualization of numerical results is an important part of FE computations. However, the current way of visualizing data from
```

Comment:
I would replace "the current way of visualizing data from ... format" by... 

"before the inception of `GridapMakie.jl`, the only approach available to data visualization of `Gridap.jl` computations was to write simulation data to data files (e.g., in vtu format) for later visualization with, e.g., Paraview or VisIt."